test/integration/suites/ghostunnel-federation/Dockerfile

```diff
@@ -3,9 +3,9 @@ FROM spire-agent:latest-local as spire-agent
 FROM ghostunnel/ghostunnel:latest AS ghostunnel-latest
 
 FROM alpine/socat:latest AS socat-ghostunnel-agent-mashup
-COPY --from=spire-agent /opt/spire/bin/spire-agent /opt/spire/bin/spire-agent
-COPY --from=ghostunnel-latest /usr/bin/ghostunnel /usr/bin/ghostunnel
-RUN apk --no-cache add dumb-init
-RUN apk --no-cache add supervisor
 ENTRYPOINT ["/usr/bin/dumb-init", "supervisord", "--nodaemon", "--configuration", "/opt/supervisord/supervisord.conf"]
 CMD []
+COPY --from=spire-agent /opt/spire/bin/spire-agent /opt/spire/bin/spire-agent
+COPY --from=ghostunnel-latest /usr/bin/ghostunnel /usr/bin/ghostunnel
+RUN apk --no-cache --update add dumb-init
```

Comment:
The --no-cache optimizations I did everywhere to reduce layer size.

This prevents the updated packages index from being persisted in the package manager dir.

Basically we don't have to do `rm -rf /var/cache/apt/*` which is another approach to reduce the image size.